<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MercuryiTC Monitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="centralwidget_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayoutTop">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <item row="2" column="3" rowspan="5">
       <spacer name="horizontalSpacer_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>4</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="t1_label">
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Temperature:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QCheckBox" name="h2_checkbox">
        <property name="text">
         <string>Automatic heater control</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLabel" name="t1_reading">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>-- K</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QLabel" name="gf3_label">
        <property name="text">
         <string>Gas flow minimum:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="ReadingDSpinBox" name="t2_edit">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string>K</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>4.000000000000000</double>
        </property>
        <property name="maximum">
         <double>300.000000000000000</double>
        </property>
        <property name="value">
         <double>290.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="7">
       <widget class="QLabel" name="labelTitle">
        <property name="font">
         <font>
          <pointsize>19</pointsize>
         </font>
        </property>
        <property name="text">
         <string>MercuryiTC Control</string>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="ReadingDSpinBox" name="h1_edit">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="ReadingDSpinBox" name="gf3_edit">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="6" rowspan="5">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="4">
       <widget class="QLabel" name="gf1_label">
        <property name="text">
         <string>Gas flow:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="t2_label">
        <property name="text">
         <string>Temperature setpoint:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="r1_label">
        <property name="text">
         <string>Ramp speed:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="7">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>2</width>
          <height>3</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="0" colspan="7">
       <widget class="QLabel" name="alarm_label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: red</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QLabel" name="h1_label">
        <property name="text">
         <string>Heater (0.0000 V):</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QCheckBox" name="gf2_checkbox">
        <property name="text">
         <string>Automatic gas flow control</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="ReadingDSpinBox" name="r1_edit">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string>K/min</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>20.000000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="ReadingDSpinBox" name="gf1_edit">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QCheckBox" name="r2_checkbox">
        <property name="text">
         <string>Temperature ramp</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QGridLayout" name="gridLayoutCanvas">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QGridLayout" name="gridLayoutBottom">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <item row="1" column="0">
       <widget class="QSlider" name="horizontalSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1440</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="timeLabel">
        <property name="text">
         <string>Show last 1 min</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>566</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_MercuryiTC">
    <property name="title">
     <string>MercuryiTC</string>
    </property>
    <addaction name="sensorAction"/>
    <addaction name="modulesAction"/>
    <addaction name="readingsAction"/>
    <addaction name="separator"/>
    <addaction name="connectAction"/>
    <addaction name="disconnectAction"/>
    <addaction name="updateAddressAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="showLogAction"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionMinimize"/>
    <addaction name="actionZoom"/>
   </widget>
   <addaction name="menu_MercuryiTC"/>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">QStatusBar{background:transparent};
QStatusBar::item {border: 0px solid black };</string>
   </property>
  </widget>
  <action name="exitAction">
   <property name="text">
    <string>&amp;Quit MercuryiTC monitor</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="sensorAction">
   <property name="text">
    <string>&amp;Temperature Sensors</string>
   </property>
  </action>
  <action name="readingsAction">
   <property name="text">
    <string>Readings Overview...</string>
   </property>
  </action>
  <action name="connectAction">
   <property name="text">
    <string>&amp;Connect Mercury</string>
   </property>
  </action>
  <action name="disconnectAction">
   <property name="text">
    <string>&amp;Disconnect Mercury</string>
   </property>
  </action>
  <action name="updateAddressAction">
   <property name="text">
    <string>&amp;Connection Settings...</string>
   </property>
  </action>
  <action name="showLogAction">
   <property name="text">
    <string> Show Log Files...</string>
   </property>
  </action>
  <action name="actionMinimize">
   <property name="text">
    <string>Minimize</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="text">
    <string>Zoom</string>
   </property>
   <property name="shortcut">
    <string>Meta+Ctrl+Z</string>
   </property>
  </action>
  <action name="modulesAction">
   <property name="text">
    <string>Assign modules...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReadingDSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>mercurygui.pyqt_labutils.scientific_spinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>r2_checkbox</tabstop>
  <tabstop>gf2_checkbox</tabstop>
  <tabstop>h2_checkbox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionMinimize</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoom</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMaximized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
